modules/mobile-agents/android/client/src/org/wso2/mdm/agent/AlreadyRegisteredActivity.java

```diff
@@ -373,9 +363,10 @@ private void initiateUnregistration(){
 		btnUnregister.setText(R.string.register_button_text);
 		btnUnregister.setTag(TAG_BTN_RE_REGISTER);
```

Comment:
Rather than encode all tiny details of the constant within the name itself, try to introduce a constant class with different inner classes upon different categories available. For example, you could have introduced something like AndroidConstants.TagInfo.BTN_RE_REGISTER or something like, which would have made the code much readable.